src/settings/TargetProfileDetail.test.js

```diff
@@ -142,10 +144,10 @@ describe('TargetProfileDetail', () => {
   });
 
   it('names of job profiles should be displayed as a hotlink', () => {
-    const { getByText } = renderTargetProfileDetail();
+    const { getAllByText } = renderTargetProfileDetail();
 
-    expect(getByText('create name').href).toContain('/settings/data-import/job-profiles/view/createJobProfileTestId');
-    expect(getByText('update name').href).toContain('/settings/data-import/job-profiles/view/updateJobProfileTestId');
+    expect(getAllByText('Test name 1')[0].href).toContain('/settings/data-import/job-profiles/view/testId1');
+    expect(getAllByText('Test name 1')[1].href).toContain('/settings/data-import/job-profiles/view/testId1');
```

Comment:
It feels like we're losing detail here: we used validate that the "create..." link corresponded to a "create..." url, ditto for "update", and now we're validating that some array elements (who knows if/how 0 and 1 are supposed to be the same or different) match. 

Is it possible to keep the level of detail we had before, or am I misunderstanding?